src/ros_filter.cpp

```diff
@@ -1017,13 +1029,12 @@ void RosFilter::loadParams()
             pose_callback_data, twist_callback_data);
 
         topic_subs_.push_back(
-          node_->create_subscription<nav_msgs::msg::Odometry>(odom_topic,
+          node_->create_subscription<nav_msgs::msg::Odometry>(odom_topic, queue_size_, 
```

Comment:
These topics have `_queue_size` parameters which should be used, like the differential / relative parameters you update below